mmpose/models/heads/heatmap_heads/heatmap_head.py

```diff
@@ -72,6 +76,7 @@ def __init__(self,
                  out_channels: int,
                  deconv_out_channels: OptIntSeq = (256, 256, 256),
                  deconv_kernel_sizes: OptIntSeq = (4, 4, 4),
+                 deconv_num_groups: OptIntSeq = (1, 1, 1),
```

Comment:
On second thought, I feel it's better to define a new class `ViPNASHead` head that inherits `HeatmapHead` and overrides the method `_make_deconv_layers`. So we don't have to make `HeatmapHead` more complex and can add a paper reference to the docstring of `ViPNASHead`.

Reply:
Changed.